### PR TITLE
Vue 1123

### PR DESCRIPTION
**Pull request by tiarascoleman**

### CODE REVIEW

File: src/pages/Settings/SavedSearchBeta.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	<www-page>
+		<kv-page-container>
+			<kv-grid class="tw-pt-4 md:tw-pt-6 lg:tw-pt-8">
+				<h1 class="tw-mb-2">
+					Saved searches
+				</h1>
+			</kv-grid>
+			<kv-grid class="md:tw-pt-6 lg:tw-pt-8">
+				<h2 class="tw-mb-1">
+					<table>
```
Comment (tiarascoleman) [inline src/pages/Settings/SavedSearchBeta.vue:11]: In progress; needs to be cleaned and will do so!

**Review by mcstover — APPROVED** [review]

File: src/pages/Settings/SavedSearchBeta.vue [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+		/* totalCount */
+	},
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			totalCount: 0,
+			savedSearches: () => []
+		};
+	},
+	/* methods : {
```
Comment (dyersituations) [inline src/pages/Settings/SavedSearchBeta.vue:47]: Do we need the two commented out sections?

File: src/pages/Settings/SavedSearchBeta.vue [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+			sortBy: BasicSortByEnum = newest
+) : SavedSearchCollection
+	}, */
+};
+
+</script>
+
+<style scoped>
+h1 {
+	font-size: 16px;
```
Comment (dyersituations) [inline src/pages/Settings/SavedSearchBeta.vue:60]: @mcstover Do we generally want to keep to the standard header sizings?

Comment (mcstover) [reply src/pages/Settings/SavedSearchBeta.vue:60]: Yes, we're cleaning all this up in subsequent PRs. This was just to get the prior work on the frame in place.

**Comment by dyersituations** [comment]
I'll approve, since there will be follow-up PRs. Looks like there are some eslint warnings in the GH view. The eslint rules recently changed a bit, so ensure eslint is running locally.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**